frontend/glideinFrontendMonitorAggregator.py

```diff
@@ -276,7 +201,7 @@ def aggregateStatus():
     for group in monitorAggregatorConfig.groups:
         # load group status file
         status_fname = os.path.join(
-            os.path.join(monitorAggregatorConfig.monitor_dir, "group_" + group), monitorAggregatorConfig.status_relname
+            monitorAggregatorConfig.monitor_dir, "group_" + group, monitorAggregatorConfig.status_relname
```

Comment:
Will it be better to change `"group_" + group` to `f"group_{group}"`?

Reply:
done